hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -226,7 +226,12 @@
       lc_states: ["TEST_UNLOCKED", "DEV", "PROD", "RMA"]
       tests: ["chip_tap_straps_dev", "chip_tap_straps_prod", "chip_tap_straps_rma",
               "chip_tap_straps_testunlock0"]
-      bazel: ["//sw/device/tests:rv_dm_jtag_tap_sel"],
+      bazel: ["//sw/device/tests:rv_dm_jtag_tap_sel_test_unlocked1",
```

Comment:
Nit: please could you make `lc_states` above match the lifecycle states here?

I'm not sure which one should be updated, @engdoreis what do you think?

Reply:
Testpoint descriptions that list `PROD` should almost always support `PROD_END` as well. It's the same as `PROD` except that it can't transition any further (e.g. to `RMA`).

Reply:
And yes, you can remove them from the `BUILD` file if they're not mentioned in the testplan.

Reply:
Latest force push has made this change.